tests/end2end/features/test_editor_bdd.py

```diff
@@ -180,18 +180,25 @@ def save_editor_wait(tmpdir):
     os.kill(pid, signal.SIGUSR2)
 
 
-@bdd.when(bdd.parsers.parse('I set up a fake "{kind}" fileselector '
-                            'selecting "{files}"'))
-def set_up_fileselector(quteproc, py_proc, kind, files):
+@bdd.when(bdd.parsers.parse('I setup a fake "{kind}" fileselector '
+                            'selecting "{files}" and writes to "{output_type}"'))
```

Comment:
```suggestion
                            'selecting "{files}" and writes to {output_type}'))
```

I typically use the quotation marks where it makes the tests easier to read (for filenames and such), but here they seem a bit confusing (arguably for `{kind}` as well, but I didn't see that when reviewing the last PR).

Reply:
Fixed, also `{kind}`